### ISSUE TITLE
Align PDF icon with the standard (no period)

### ISSUE THREAD

**Issue by thepegisin** [cagov/drought.ca.gov#219]
While documenting engineering components for our Design System, I noticed that the PDF icon used on Drought differs from the standard one used on COVID-19 and in the Design System.

It's the content team's consensus that this icon should be just the letters PDF contained in a box. There should be no period before the letters.

**Comment by kkoryaka** [comment]
I will have to update design system component. Will try to update it this week

**Comment by xjensen** [comment]
Quick note. We have a related Issue at #221.

**Comment by kkoryaka** [comment]
@thepegisin please review in staging: https://staging.drought.ca.gov/
